core/src/main/scala/io/chrisdavenport/rediculous/RedisCommands.scala

```diff
@@ -361,11 +361,15 @@ object RedisCommands {
     RedisCtx[F].unkeyed(NEL("XREVRANGE", stream :: end ::: start ::: count))
   }
 
-  def xgroupcreate[F[_]: RedisCtx](stream: String, groupName: String, startId: String, mkStream: Boolean = false): F[Status] = {
+  def xgroupcreate[F[_]: RedisCtx](stream: String, groupName: String, startId: String, mkStream: Boolean): F[Status] = {
     val mkStreamFragment = Alternative[List].guard(mkStream).as("MKSTREAM")
     RedisCtx[F].unkeyed(NEL("XGROUP", "CREATE" :: stream :: groupName :: startId :: mkStreamFragment))
   }
 
+  @deprecated("use xgroupcreate(stream: String, groupName: String, startId: String, mkStream: Boolean) instead", "0.3.3")
+  def xgroupcreate[F[_]: RedisCtx](stream: String, groupName: String, startId: String): F[Status] = 
```

Comment:
I recommend you change the signature like this, so that it is source-breaking and discourages accidental use.

```suggestion
  def xgroupcreate[F[_]](stream: String, groupName: String, startId: String, ctx: RedisCtx[F]): F[Status] = 
```

Reply:
Interesting I thought you made a mistake there. 😸

Reply:
I don't make mistakes 😝

Reply:
(except about the order of args in `@deprecated` I guess 😂 )

Reply:
Should ctx be implicit?

Reply:
No, you can pass it explicitly. The point of this unusual signature is to maintain binary-compatibility but break-source-compatibility by making it very difficult to use.

Reply:
Thats a nice little trick I learnt. Thank you. 😄